Core/iTasks.php

```diff
@@ -0,0 +1,264 @@
+<?php
+/**
+ * @file
```

Comment:
This file has mostly profile related functions. I think most of this should still be maintained in the profile itself. 

The itasks_includes() function might be one to keep here as it is relative to the classes but the profile alters can probably stay in the profile.